### ISSUE TITLE
Please make it easier to manage skip domains

### ISSUE THREAD

**Issue by foresto** [diegocr/CleanLinks#95]
I have been using CleanLinks for just a couple weeks, and already my Skip Domains list is hundreds of characters long. The fact that they are all stuffed into a single line in a tiny text box makes it a real pain to manage. It would be great if there were a UI for easily viewing and editing the list. Even better if it were accessible directly from the toolbar button (like every other domain-related add-on I use) rather than having to find and deal with Firefox's Add-Ons menu.

**Comment by rosmcmahon** [comment]
Yes, that tiny box just emptied itself on my android tablet. Now I have nothing in there. Extremely awkward
